Makefile

```diff
@@ -7,7 +7,14 @@ DESTDIR ?=
 SIGN ?=
 PREFIX ?= /usr/local
 FLAKY_TESTS ?= run
+STAGINGSERVER ?= node-www
 
+OSTYPE := $(shell uname -s | tr '[A-Z]' '[a-z]')
+
+# Flags for packaging.
+BUILD_DOWNLOAD_FLAGS ?= --download=all
```

Comment:
/cc @srl295, I agree.

But, it's not really relevant to this PR as it's already on `master`, `v5.x` and `v4.x`, a change should filter down from there